feat:example page and hot update

lucas200631 opened the pull request:
<!-- 非常感谢您的对本项目的 PR 和贡献。-->

## Types of changes
<!-- 请为本次PR选择一种类型，如果有多个类型，您可以在 Changelog 中添加"type"列。-->
<!-- 在 "[ ]" 中输入“x”以勾选选框) -->

- [x] New feature
- [ ] Bug fix
- [ ] Enhancement
- [ ] Component style change
- [ ] Typescript definition change
- [ ] Documentation change
- [ ] Coding style change
- [ ] Refactoring
- [ ] Test cases
- [ ] Continuous integration
- [ ] Breaking change
- [ ] Others

## Background and context
为使用者提供一个example页面, 可以在跑起项目的同时打开index.html文件
并且实现了服务热更新 当源码有改动时 打包文件自动更新
<!-- 解释这个PR解决了什么问题 -->


<!-- 如果有相关的issuse请关联上 -->

## Solution

<!-- 详细描述如何解决问题 -->


## Changelog

|   type    | Changelog(CN) | Changelog(EN) | Related issues |
| --------- | ------------- | ------------- | -------------- |
|           |               |               |                |


## Other information

<!-- 此处用来填写补充更多的信息，例如破坏性更新的影响、效果图等。 -->

Commit:
feat:example page and hot update

oljc commented on package.json:
```diff
@@ -65,5 +68,9 @@
       "pre-commit": "lint-staged",
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
+  },
```
This dependency package should be placed in devDependencies

oljc commented on rollup.config.js:
```diff
@@ -3,6 +3,8 @@ import strip from '@rollup/plugin-strip';
 import terser from '@rollup/plugin-terser';
 import filesize from 'rollup-plugin-filesize';
 import typescript from '@rollup/plugin-typescript';
+import serve from 'rollup-plugin-serve';
```
This should be separate into a separate development rolllup

oljc commented on example/index.html:
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```
There is no default usage

Commit:
feat:example page and hot update

oljc committed:
Remove 'type: module' from the package.json file

This commit removes the "type: module" configuration from the package.json file. This configuration is used to indicate that a module is using ES6 module syntax instead of CommonJS syntax in Node.js. By removing this configuration, the module will default to using CommonJS syntax.

This change may be necessary if the module was previously using ES6 module syntax but needs to be compatible with older versions of Node.js or other environments that do not support it. Additionally, removing this configuration may improve the module's compatibility with other tools and libraries that are designed to work with CommonJS modules.
